ovs_performance.py

```diff
@@ -4102,6 +4121,7 @@ def main():
     #                     "test_p2v_1000")
     # sys.exit(-1)
 
+
```

Comment:
Can you remove this extra line you introduced?

Reply:
NP